includes/widgets/common.php

```diff
@@ -1727,8 +1150,6 @@ protected function register_controls() {
 
 		$this->register_effects_section();
 
-		$this->register_transform_section();
```

Comment:
is this removing the transform from other widgets?

Reply:
@TzviRabinovitch You were right. I have removed this part of the code.